src/pages/gd/Claim/ClaimBalance.tsx

```diff
@@ -40,14 +40,15 @@ export const ClaimBalance = () => {
                 .catch(() => undefined),
         [chainId]
     )
-    const formattedTime = useMemo(() => format(claimTime, 'hh aaa'), [claimTime])
+
+    const formattedTime = useMemo(() => claimTime && format(claimTime, 'hh aaa'), [claimTime])
     const { switchNetwork } = useEthers()
 
-    const network = useMemo(() => SupportedChains[chainId], [chainId])
+    const network = chainId === (SupportedChains.FUSE as number) ? SupportedChains[42220] : SupportedChains[122]
```

Comment:
why not just SupportedChains[chainId] ? you haven't answered

Reply:
Because we are requesting the opposite chain, not the current chain

Reply:
add a code comment about it